### ISSUE TITLE
SBC's including Raspberry Pi 4 not good enough to run motioneye with high frame rates

### ISSUE THREAD

**Issue by bobster316** [ccrisan/motioneyeos#2100]
I am trying to get my Wyze Cam v2's to run smooth frame rates on motioneyeos. I have tried Raspberry Pi 3B. Raspberry Pi 4B 4Mb, Odroid XU4. All of these are not good enough to run smooth frame rates on motioneye. 
I have the SBC's connected via Ethernet directly to my Netgear Nighthawk R9000 and the Wyze Cam's connected via WiFi. 
I have overclocked my Raspberry Pi 4B to 2Mhz and GPU to 620Mhz but the results are still not good enough. Approx 0.5/8.7 Fps according to motioneye.
What are my options? I do not want to drop from 1080p to lower..

**Comment by jhaury** [comment]
I don't think it has anything to do with motioneye or SBCs. I have been using motioneyeos for the past few weeks on a RPi3 B+ streaming a RPi Zero W/camera and a Wyze Cam 2. The Wyze cams do not support rtsp out of the box, you must use their beta firmware or the hacked one to get the feature. The fps I get from the wyze cam over rtsp is bad, it is the same if I am watching the feed from my Mac Pro. The RPi zero cam performs better, faster framerates.

The wyze cam has no settings that I am aware of to change its output or framerate for streaming, other than 1080/720 option in the app. I think the issue is the wyze cameras not motioneye or the SBCs. Thats my experience.

**Comment by Chiny91** [comment]
I’m using the lower res 720p on rtsp for 3 cams on one R Pi3+.  I found big cpu guzzlers include (1) not matching the cam res out, to the motionEyeOS video res in, under video device, and (2) having more than two “captured before” pics.   ISTR (can’t check right now) I get the required 4fps, although probably only on one cam at a time (all I require).   Everything not used is switched off on the R Pi, pics are nice’d & rsync’d elsewhere for producing video, all to ease the strain on the R Pi.   Maybe something there to assist your investigation.
